### PR TITLE
Add first CfP version

### PR DESCRIPTION
**Pull request by jossiwolf**
This is a first proposal for the CfP site. Note that this is just the content and no dedicated page for now.

### CODE REVIEW

File: data/cfp.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+💻 Web (Polymer, PWAs, Angular, Dart, new standards, tooling, ...)  
+☁️ Cloud (TensorFlow, big data, Kubernetes, security, ...)  
+📟 IoT (Android Things, Actions API, Android Wear, Chromecast, Arduino, ...)  
+🌏 Science (physics, universe, space travel, future, ...)  
+💼 Marketing & business skills targeted at indie developers or startups  
+🙋 Soft skills for developers
+
+To give you a better idea of what topics were talked about before, take a look at the sessions from the following DevFests:
+
+* [GDG DevFest Hamburg 2017](https://hamburg.devfest.de/schedule/day1)
```
Comment (jossiwolf) [inline data/cfp.md]: If you know more cool conferences that can give speakers a better idea of the content we aim for, feel free to add them!

File: data/cfp.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+### Guidelines 📃
+
+-   Language: all talks are in English.
+-   Formatting: make sure you care, and make sure we see you care. Typos, sloppy formatting and all-lowercase submissions make our reading of your proposal tedious and will reflect negatively on the outcome of your submission.
+-   Brevity: don’t get too wordy. If you need more than two paragraphs to get to the point of your proposal, we ask you to put more time into focussing your application. With the amount of submissions we get, the quicker you can make a good first impression, the better.
+- We are not looking for talks that were recorded before or given to an audience bigger than 200 people. You don't need to submit a completely new talk, but could for example add a twist or new part to a pre-existing talk.
+
+### Talk Formats 🗣️
+   
+We are looking for two types of sessions:
```
Comment (jossiwolf) [inline data/cfp.md]: I just used what could make sense, totally up for discussion!

Comment (jossiwolf) [inline data/cfp.md]: Should we add workshops?

File: data/cfp.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
+## DACHFest 2018 Call for Papers
+
+Thanks for your interest in speaking at DACHFest 2018! DACHFest is a community-driven conference organised by all GDGs (Google Developer Groups) in Germany, Austria and Switzerland that will take place from 10. Nov 2018 to 11. Nov 2018 at Technical University Munich in Munich, Germany. We expect about 800 attendees per day with 3 simultaneous tracks.
```
Comment (thebino) [inline data/cfp.md]: "Technical University Munich in Munich" redundancy is not needed here

File: data/cfp.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+If you have any questions, do not hesitate to contact us on Twitter or at [cfp@dachfest.com](mailto:cfp@dachfest.com).
```
Comment (thebino) [inline data/cfp.md]: Add twitter handle "@dachfest"

File: data/cfp.md (hunk 3 above)
Comment (thebino) [inline data/cfp.md]: "Technical University Munich in Munich" redundancy is not needed here

**Comment by jossiwolf** [comment]
All done!

**Comment by emedvedev** [comment]
Ok, that was a lot of changes, and I should've really submitted a PR, but worst case we can just revert. @jossiwolf please take a look. 

Some changes:

* CFP dates: this is what I'd like to be final.
* The "topics" section: there was no mention of our main theme, which is really the main guideline for what we want to see at the conference.
* The "guidelines" section: was a bit too strict to my taste, but that's subjective, of course, so we can revert.
* Added a "short talk" format and an encouragement to apply with other formats/lengths.
* Added our amazing mentors (❤️) to the "Support" section. TODO: Madlaina's email.

Other changes are pretty much just shuffling words around, but if you feel like something got lost in the process, please let me know.

**Comment by jossiwolf** [comment]
@emedvedev Your writing is beyond anything I have ever read before. Awesome work!

About the topics: I think we should keep the non-exhaustive list of topics as it gives people a better idea of what the general frame is. But of course, that decreases the number of tasks outside the topics on this list. Maybe we can mention to contact us if there are any questions about the topics? What do you think?

**Comment by emedvedev** [comment]
Aw schucks, thanks for the compliment.

I've added a paragraph that tells people to contact CFP mentors or @dachfest on Twitter if things aren't super clear. I really really want to keep the theme broad and avoid lists (I don't even like the examples that are there now): no matter how clear we make the fact that the list is non-exhaustive, some people will hesitate to submit if their talk isn't on it. It's a thing with job requirements, too: a lot of people, especially from underrepresented groups, are hesitant if they see they don't check every single mark on the list, even if some of the marks are completely optional.

**Comment by jossiwolf** [comment]
@emedvedev Perfect! Also, I love the naming of talk formats!

How do we proceed with this? Should we just merge it so whoever is setting up Papercall can access it?

**Comment by emedvedev** [comment]
Let's wait on @AnnKatrinfromGDG and @Aadjou to review this as well, and then I'll set up Papercall. We can still merge just so we have a backup version here. :)

File: data/cfp.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+DACHFest is a community-driven conference organized by Google Developer Groups from Germany, Austria, and Switzerland. It's taking place at the Munich Technical University on November 10–11: around 800 people are going to get together to enjoy 3 talk tracks, a lot of community-oriented activities, and opportunities for networking and making new friends. ❤️
+
+If you have any questions, do not hesitate to contact us on [Twitter](https://twitter.com/DACHFest) or at [cfp@dachfest.com](mailto:cfp@dachfest.com).
+
+### Deadlines 📅
+
+September 15: Last day to submit your application!
+
+September 30: We'll notify you of your submission status.
```
Comment (Aadjou) [inline data/cfp.md]: 15 days is extremely tight. do we have committed volunteers for the review process in this time?
I suggest moving the deadline to 1.09 to give a little bit more time

File: data/cfp.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+We strive for eliminating bias as much as possible, and it means that every submission will be anonymized and evaluated entirely on its merits. We are not looking for hot startups and famous names — we are looking for exciting stories and interesting people.
+
+No matter whether your talk is selected or not, we will always try to provide feedback and give you suggestions if we have any.
+
+### Guidelines 📃
+
+* **Language**: English.
+* **Brevity**: we recommend a single paragraph for the abstract and 3-4 paragraphs for the talk description. Get straight to the point!
+* **Formatting and proofreading**: submissions with good spelling, no typos, and proper paragraph formatting make the job of our program committee much easier.
+* **Completeness**: you will not have a chance to modify your submission! Read it carefully before sending, make sure it's complete and has all the information we need to make an objective decision.
```
Comment (Aadjou) [inline data/cfp.md]: + Ask someone to review your abstract before submission. Our mentors are also happy to guide you and give valuable advice for your entry.

Comment (Aadjou) [reply data/cfp.md]: i see now that it is below. but maybe it's worth mentioning here as well

File: data/cfp.md [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+Nobody is a born speaker. If you want to submit a talk, you probably have something exciting to tell the world — and that's enough for us!
+
+If you feel like you could use some help with your submission, reach out to one of our amazing mentors:
+
+* Madlaina Kalunder ([@anialdam](https://twitter.com/anialdam) or [madlaina@dachfest.com](mailto:madlaina@dachfest.com))
+* Martin Splitt ([@g33konaut](https://twitter.com/g33konaut) or [splitti@google.com](mailto:splitti@google.com))
+
+They are here to give you all the support you might need. Here's a non-exhaustive list of things they could help with:
+
+* Finding a good talk topic
```
Comment (Aadjou) [inline data/cfp.md]: Boosting your confidence

File: data/cfp.md (hunk 5 above)
Comment (emedvedev) [reply data/cfp.md]: I think that's the only way we can swing it. If we move the deadline to September 1, we won't have enough time to course-correct if necessary, and if we move notifications any further, we might mess our speakers' schedule or even have some people double-book.

File: data/cfp.md (hunk 6 above)
Comment (emedvedev) [reply data/cfp.md]: Done

File: data/cfp.md (hunk 7 above)
Comment (emedvedev) [reply data/cfp.md]: Done

**Comment by jossiwolf** [comment]
Closing as CfP is live now 🎉
